core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -493,6 +493,18 @@ public BooleanColumn set(Selection rowSelection, boolean newValue) {
     return this;
   }
 
+  @Override
+  public Column<Boolean> setObj(int row, Object obj) {
+    if (obj == null) {
+      return setMissing(row);
+    }
+    if (!(obj instanceof Boolean)) {
+      throw new IllegalArgumentException(
+          "Cannot append " + obj.getClass().getName() + " to BooleanColumn");
```

Comment:
"append" might not be quite the right word since we're not in the `append` method. Maybe "add" or something else?